.github/issue_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- IF YOU DO NOT FILL THIS OUT, WE WILL CLOSE YOUR ISSUE! -->
+
+**I'm submitting a ...**  (check one with "x")
+```
+[ ] bug report => search github for a similar issue or PR before submitting
+[ ] feature request
+[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
+```
+
+**Current behavior**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Reproduction of the problem**
+<!-- If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/vtAo2zzWFn8kFOGRH88r). -->
```

Comment:
plnkr is for javacript. But an example of code is great. A Github repository of a reproducer would be nice instead.